doc/Building_targets.md

```diff
@@ -0,0 +1,179 @@
+# Building targets
+
+KOReader is available for multiple platforms. Here are instructions to build installable packages for all these platforms.
+
+These instructions are intended for a Linux OS. MacOS and Windows users are suggested to develop in a Linux VM.
+
+
+## Prerequisites
+
+This instructions asume that you [have a development environment ready to run](Building.md) KOReader. If not then please install common prerequisites first.
```

Comment:
There's a premade Docker image available for every supported target.

Reply:
I would need a bit of help with that since I was unaware and I never used Docker beyond a few tests.

Reply:
You mean to mention specific commands in this file? That's possible too. I was basically just thinking of referencing the other README file in the virdevenv repo, which could use some more expanding. Currently more or less presumes you know Docker.

You need Git & Docker, or even just Docker will do. Assuming you want to interact with the koreader dir from your regular environment (text editor etc., but normally also git-related commands) you run:

```bash
docker run -v $(pwd)/koreader:/home/ko/koreader -it koreader/koappimage:latest bash
# Now you're in Bash in the Docker image, and commands like ./kodev build work out of the box
# The Docker image includes Git, hence why you don't need it on the host
# So besides the docker run part everything is the same as in the guide here.
# Docker gives you ready-made prerequisites.
```

Where "koappimage" is whatever target you're looking for. The appimage one will take care of all your regular desktop needs. Actually the baseimage can already do Debian packages and basic emulator; you'd need to run it on the host with SDL installed in that case. Only the AppImage one can run the emulator *from* Docker.

The `-v` flag mounts your local directory in the Docker container when it runs. This isn't required, but otherwise Docker tosses out all changes (i.e., downloads & compiled binaries) on exit. 

I hope that clarifies things a little bit.